### ISSUE TITLE
[canal] add a skip prepareDumper() logic

### ISSUE THREAD

**Issue by zhujintao** [go-mysql-org/go-mysql#922]
`exec: "mysqldump": executable file not found in $PATH`

```
	if err = c.prepareDumper(); err != nil {
		return nil, errors.Trace(err)
	}

```
use custom full import, non mysqldump
add skipDump logic:
```
	if !c.cfg.skipDump {
		if err = c.prepareDumper(); err != nil {
			return nil, errors.Trace(err)
		}
	}
```

**Comment by rockwotj** [comment]
Yeah it would be cool to replicate what mysqldump does in go instead of shelling out

**Comment by dveeden** [comment]
I think adding a `skipDump` option could be useful. Replacing `mysqldump`/`mariadb-dump` would be welcome, but it is also a lot of work to get that working correctly.

**Comment by equnchen** [comment]
![image](https://github.com/user-attachments/assets/1c463a97-2a75-4230-9eeb-eabebf306a0a)
There is already a  option,  please use c.cfg.Dump.ExecutionPath="" @zhujintao

**Comment by zhujintao** [comment]
@equnchen ok,thank you!
